server/routes/api/chart.py

```diff
@@ -390,24 +390,22 @@ def get_choropleth_sv():
             dictionary of stat var: list of denominators
     """
     chart_config = current_app.config['CHART_CONFIG']
-    if os.environ.get('FLASK_ENV') == 'development':
-        with bp.open_resource('../../chart_config.json') as f:
-            raw_config = json.load(f)
     all_sv = set()
     sv_denom_mapping = {}
     for config in chart_config:
-        if 'hasChoropleth' in config and config['hasChoropleth']:
+        if config.get('hasChoropleth', False):
             # we should only be making choropleths for configs with a single stat var
+            # TODO(chejennifer) add test for chart config to ensure hasChoropleth is only added to charts with single statvar
             sv = config['statsVars'][0]
             all_sv.add(sv)
             denom = ''
-            if 'denominator' in config:
+            if config.get('relatedChart',
+                          {}) and config['relatedChart']['scale']:
```

Comment:
can also do config['relatedChart'].get('scale', False) incase the 'scale' field is not set

Reply:
and also if config.relatedChart.scale == True, but config.related.denominator is not set, it defaults to "Count_Person"